packages/amplication-server/src/core/resource/resourceBtm.service.spec.ts

```diff
@@ -324,11 +345,17 @@ describe("ResourceBtmService", () => {
             functionality: "manage orders, prices and payments",
             tables: ["order", "orderItem"],
           },
+          {
```

Comment:
Review this test as the title and the expectations are not aligned

Reply:
https://github.com/amplication/amplication/pull/8008/commits/25259309dccd5010d780844262e478c4b40cf1e9